### PR TITLE
Development

### PR DESCRIPTION
**Pull request by aklim**
Change namespace according repository name.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-The "borales/yii2-phone-input" is free software.
+The "aklim/yii2-phone-input" is free software.
 It is released under the terms of the following BSD License.
 
-Copyright © 2014 by Alexandr Bordun. All rights reserved.
+Copyright © 2017 by Andrey Klimenko. All rights reserved.
```
Comment (Borales) [inline LICENSE.md]: @aklim I don't think stealing someone's code and changing the copyright is a good idea

**Comment by aklim** [comment]
Sorry, this is my fault. I'm apply it with auto complete. I remove this
repository and don't touch original code.

Regards,
Andrey.

14 окт. 2017 г. 14:47 пользователь "Oleksandr Bordun" <
notifications@github.com> написал:

**Comment by Borales** [comment]
@aklim you can submit your pull-request to this repository with any improvements you want, but you shouldn't change namespace and license copyright holder.

The same applies to this: https://github.com/aklim/yii2-phone-input

**Comment by aklim** [comment]
OK, this repository removed.
I just don’t understand how to use it (countryCode doest not work in latest version), so I need to copy it some there.

Against, I apologize for this mistake.

Best Regards,
Andrey.

**Pull request closed without merging**